Platform/Plugins/com.tle.platform.common/src/com/tle/common/util/ExecUtils.java

```diff
@@ -114,6 +115,38 @@ public static ExecResult exec(String[] cmdarray, Map<String, String> additionalE
     }
   }
 
+  public static ExecResult execWithTimeLimit(long maxDurationInSeconds, String[] options) {
+    if (maxDurationInSeconds < 1L) {
+      LOGGER.info("maxDurationInSeconds not set. Using a regular non-timed process.");
```

Comment:
This will make for some rather verbose logs, and is really only relevant for debugging issues so

```suggestion
      LOGGER.debug("maxDurationInSeconds not set. Using a regular non-timed process.");
```